admin/clientconfig.go

```diff
@@ -20,9 +20,10 @@ const (
 )
 
 type ProtocolConfig struct {
```

Comment:
it's probably good to also mention in the client configs (as shown in the quickstart with printClientConfigs from main.go, and as shown in the admin web interface on the domain page under "Client settings") when a service is also available over port 443 and with which ALPN value. see ClientConfigsDomain later on in this file. after "with TLS" we can probably just add `; also served on port 443 with TLS ALPN "smtp"`.

Reply:
How's this?

```
s0ph0s@lima-default:/home/mox$ sudo ~mox/mox clientconfig chatmail.s0ph0s.dog
Protocol             Host                            Port Listener        Note
Submission (SMTP)    mail.chatmail.s0ph0s.dog         465 public          with TLS; also served on port 443 with TLS ALPN "smtp"
IMAP                 mail.chatmail.s0ph0s.dog         993 public          with TLS; also served on port 443 with TLS ALPN "imap"

To prevent authentication mechanism downgrade attempts that may result in
clients sending plain text passwords to a MitM, clients should always be
explicitly configured with the most secure authentication mechanism supported,
the first of: SCRAM-SHA-256-PLUS, SCRAM-SHA-1-PLUS, SCRAM-SHA-256, SCRAM-SHA-1,
CRAM-MD5.
```